src/lib/components/event/event-empty-row.svelte

```diff
@@ -6,7 +6,7 @@
 </script>
 
 <tr class="row">
-  <td class="table-cell" colspan="6">
+  <td colspan="6">
```

Comment:
I have removed classes like `table-cell`, etc. since they are default styling for the element.